app/views/dashboards/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="row">
   <div class="col-sm-12">
-    <%= render partial: 'dashboards/overview' %>
+    <% unless Config.hide_budget_bar? %>
```

Comment:
Again not necessary, but we tend to do browser selenium tests for stuff like this (rails calls em system tests). There's probably an example to crib off in `test/system/updating_configs_test.rb`. That said, I wouldn't bother with this unless you've written browser tests before, as they're kinda frustrating to troubleshoot.